src/Sarif.UnitTests/Core/StackTests.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public class StackTests
     {
-        [Fact]
+        [Fact(Skip ="Broken by new, unexpected framework behavior in StackTrace class")]
```

Comment:
>Skip [](start = 14, length = 4)

Where's the issue on this? We should be referencing it in a code comment for any disabled test.